pkg/apis/nodeup/config.go

```diff
@@ -90,6 +90,8 @@ type BootConfig struct {
 	ConfigBase *string `json:",omitempty"`
 	// ConfigServer holds the configuration for the configuration server.
 	ConfigServer *ConfigServerOptions `json:",omitempty"`
+	// MasterInternalName is the API server name/address.
+	MasterInternalName string
```

Comment:
I like this... if we're adding a field, should we (1) use the non-deprecated naming and (2) be clearer about whether it is apiserver or kops-controller?

How about `APIServerInternalName` ?

Reply:
Initially the intention was to keep it as close as possible to the original name. Though, I think `APISever` should be good enough. Thanks for the suggestion.